packages/utils/src/worldwide.ts

```diff
@@ -44,34 +68,23 @@ export interface InternalGlobal {
    */
   _sentryDebugIds?: Record<string, string>;
   __SENTRY__: {
-    hub: any;
-    logger: any;
-    extensions?: {
-      /** Extension methods for the hub, which are bound to the current Hub instance */
-      // eslint-disable-next-line @typescript-eslint/ban-types
-      [key: string]: Function;
-    };
-    globalScope: Scope | undefined;
-    defaultCurrentScope: Scope | undefined;
-    defaultIsolationScope: Scope | undefined;
-    globalMetricsAggregators: WeakMap<Client, MetricsAggregator> | undefined;
-    /** Overwrites TextEncoder used in `@sentry/utils`, need for `react-native@0.73` and older */
-    encodePolyfill?: (input: string) => Uint8Array;
-    /** Overwrites TextDecoder used in `@sentry/utils`, need for `react-native@0.73` and older */
-    decodePolyfill?: (input: Uint8Array) => string;
-  };
+    [key: VersionString]: BackwardsCompatibleSentryCarrier;
```

Comment:
m: Can we make a `SentryCarrier` that is the up-to-date thing and use this here, and have `BackwardsCompatibleSentryCarrier` extend from this (and use it for the "global" form)?

Reply:
We can, mostly. What we can't do is deduplicate the already existing `SentryCarrier` in Core because it relies on types from `@sentry/core` and `@sentry/types`. So we'd still type most properties as `any` here

Reply:
Done! I also marked a couple of properties to remove in v9 in the respective `@sentry/types` version of `SentryCarrier`.